odl/tomo/backends/astra_cuda.py

```diff
@@ -374,7 +375,10 @@ def astra_cuda_bp_scaling_factor(proj_space, reco_space, geometry):
     scaling_factor /= (reco_space.weighting.const /
                        reco_space.cell_volume)
 
-    if parse_version(ASTRA_VERSION) < parse_version('1.8rc1'):
+    if parse_version(ASTRA_VERSION) >= parse_version('1.9.9.dev'):
+        scaling_factor /= float(reco_space.cell_volume)
+        scaling_factor *= float(geometry.det_partition.cell_volume)
+    elif parse_version(ASTRA_VERSION) < parse_version('1.8rc1'):
```

Comment:
Prefer consistent directions (e.g. `>` everywhere or `<` everywhere), it's a bit hard to follow the if statements atm.

Reply:
Fair enough. I've added a separate commit to make things chronological.

I didn't do this before because the version currently in master is non-chronological, which made for a confusing diff. This separate commit to clean this up is hopefully the best of both worlds.